packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -42,6 +50,31 @@ const FlexButton = styled(Flex).attrs({ as: 'button' })`
   }
 `;
 
+const SelectWrapper = styled(Box)`
+  // Styling changes to SingleSelect component doesn't work, so adding wrapper to target SingleSelect
```

Comment:
just grammar but changes is plural so
```suggestion
  // Styling changes to SingleSelect component don't work, so adding wrapper to target SingleSelect
```